leo/span/src/symbol.rs

```diff
@@ -131,6 +131,7 @@ symbols! {
     main,
     Mut: "mut",
     prelude,
+    Public: "public",
```

Comment:
N.B. you only need to do this if the RHS is a keyword in rust or isn't otherwise a valid identifier.

Reply:
Got it I'll change that!